odl/operator/fn_ops.py

```diff
@@ -0,0 +1,406 @@
+## Copyright 2014-2016 The ODL development group
+##
+## This file is part of ODL.
+##
+## ODL is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## ODL is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+#
+#"""Default operators defined on any (reasonable) space."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+from odl.operator.operator import Operator
+from odl.space import fn
+from odl.set import LinearSpace, Field
+#from odl.util import writable_array
+
+import numpy as np
+
+__all__ = ('SamplingOperator', 'ZerofillingOperator', 
+           'VectoriseOperator', 'EmbeddingOperator')
+
+class SamplingOperator(Operator):
+
+    """Operator that samples coefficients.
+
+        ``SamplingOperator(f)(i) == cell_volume * f(i) \approx \int_cell f(x) dx``
```

Comment:
Use math typesetting or equation typsetting, try to mix. I.e

```
.. math::
    SamplingOperator(f)_i = cell_volume * f_{indices_i} \approx \int_{\text{cell}_{indices_i}} f(x) dx
```
    
OR

```
The operator is defined by::
    SamplingOperator(f, indices) == cell_volume * f[indices]
```

I would suggest the second option myself.

Further, I would personally prefer if this operator did not scale by `cell_volume`, i.e.

```
The operator is defined by::
    SamplingOperator(f, indices) == f[indices]
```